src/DashboardService.Models.Dto/Requests/Group/CreateGroupRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Group;
+
+public record CreateGroupRequest
+{
+  public Guid BoardId { get; set; }
+  public string GroupName { get; set; }
+  public bool GroupIsActive { get; set; }
```

Comment:
if you create a group - it is active 100%. no need in this property

Reply:
++